package.json

```diff
@@ -91,8 +91,10 @@
     "prerender:cds": "cross-env SPARTACUS_CDS=true ng run storefrontapp:prerender",
     "release:cds:with-changelog": "cd projects/cds && release-it && cd ../..",
     "build:myaccount": "ng build my-account --prod",
-    "release:myaccount:with-changelog": "cd feature-libs/my-account && release-it && cd ../..",
+    "release-notes:cds": "ts-node ./scripts/changelog.ts --verbose --lib cds",
```

Comment:
`release-notes:cds` in MyOrg PR looks strange.

Reply:
@marlass can you help with this one?

Reply:
In release config file it was already updated. Removing this no longer used script.